api-spec/event-dispatcher-api.yaml

```
@@ -0,0 +1,194 @@
+openapi: 3.0.0
+info:
+  version: 0.0.1
+  title: Pagopa eCommerce event dispatcher service
+  description: This microservice handles transaction's lifecycle and workflow processing async events.
+  contact:
+    name: pagoPA - Touchpoints team
+tags:
+  - name: event-receiver-controller
+    description: Api's for handle events receivers status
+    externalDocs:
+      url: https://pagopa.atlassian.net/wiki/spaces/I/pages/664076771/-servizio+event+dispatcher
+      description: Technical specifications
+externalDocs:
+  url: https://pagopa.atlassian.net/wiki/spaces/I/pages/492339720/pagoPA+eCommerce+Design+Review
+  description: Design review
+servers:
+  - url: https://api.platform.pagopa.it/event-dispatcher
+paths:
+  /event-receivers/commands:
```

Naming: I'd move this under `status` as it seems to me to be more REST-y

```suggestion
  /event-receivers/status:
```

---

I name this as `commands` because this api is used to create a new command that will be applied to all event receivers. I would not add this api under status path just because here no status is created.
Idk about `update` as path either. 🤔 
I will leave it as command just for the above reason.